src/App.js

```diff
@@ -1,24 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import './fonts/Hind-Regular.ttf'
+import './fonts/Montserrat-VariableFont_wght.ttf'
+import Header from './components/Header'; 
+import Form from './components/Form';
+import Gallery from './components/Gallery'
+import Footer from './components/Footer';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+
+  const [recipes, setRecipes] = useState([]);
```

Comment:
Might want to think about creating a main component to host all the state, because every time App.js refreshes it re-renders everything including its children.